src/main/java/dk/sdu/se_f22/brandmodule/index/BrandIndex.java

```diff
@@ -17,7 +21,34 @@
 
     @Override
     public void indexBrandInformation(Brand brand, List<String> tokens) {
-
+        try {
+            PreparedStatement mapInsert = DBConn.prepareStatement("INSERT INTO tokenbrandmap(brandid, tokenid) VALUES (?,?)");
+            PreparedStatement tokenInsert = DBConn.prepareStatement("INSERT INTO tokens(token) VALUES (?)");
+            PreparedStatement queryToken = DBConn.prepareStatement("SELECT token FROM tokens");
+
+            ResultSet rs = queryToken.executeQuery();
+
+            ArrayList<String> newTokens = new ArrayList<>(tokens);
+
+            for (int i = 0; i < tokens.size(); i++){
```

Comment:
Isn't a c-style loop a little excessive if you only have one value to parse into a the statement?

Reply:
we dont think its to excessive for this size